### PR TITLE
Add bundler/inline and install webrick locally to run tests

### LINKED ISSUE #27: webrick is no longer a standard ruby gem, tests failing

### PR DESCRIPTION
**Pull request by zenspider**
Closes #27

### CODE REVIEW

**Comment by zenspider** [comment]
also happy to dress this up to work w/o bundler if webrick IS available (tho that'll be decreasing over time)

**Comment by eradman** [comment]
Does Ruby ship with a simple HTTP server? I would prefer to use something built-in. I spent some time searching for a solution before, but didn't find any good answers.

**Comment by zenspider** [comment]
It **_was_** webrick. First as part of the stdlib... then as a gem bundled. It got removed in a recent version of ruby... I think it is still considered the "defacto" web server for ruby because it is pure ruby and so well known... so I think flipping to bundler to pull it back in isn't the worst.

File: tests/test_rinstall.rb [hunk 1]
```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+ENV["GEM_HOME"] ||= "tmp"
```
Comment (eradman) [inline tests/test_rinstall.rb:4]: Let's make this `.gem`

File: input.c [hunk 2]
```diff
@@ -81,7 +81,7 @@ yylex() {
 				exit(1);
 			}
 			strlcpy(tmp_src, "/tmp/rset_local.XXXXXX", sizeof tmp_src);
-			mktemp(tmp_src);
+			mkstemp(tmp_src);
```
Comment (eradman) [inline input.c:84]: Good. Unrelated, but since the directory is created locally this is correct.

File: .gitignore [hunk 3]
```diff
@@ -14,3 +14,4 @@ tests/getsocket
 tests/parser
 tests/ssh_command
 tests/which
+tests/tmp
```
Comment (eradman) [inline .gitignore:17]: tests/.gem

**Review by eradman — CHANGES_REQUESTED** [review]

**Pull request merged**